beets/util/__init__.py

```diff
@@ -794,9 +798,15 @@ def convert(arg):
     return [convert(a) for a in args]
 
 
+# stdout and stderr as bytes
+CommandOutput = namedtuple("CommandOutput", ("stdout", "stderr"))
+
+
 def command_output(cmd, shell=False):
     """Runs the command and returns its output after it has exited.
 
+    Returns a CommandOutput.
```

Comment:
Might be nice to mention here, just for convenience, that you can access `stdout` and `stderr` in the result object, both of which are bytestrings.

Reply:
Done.

One can also access the byte strings via deconstruction:
```python
stdout, stderr = CommandOutput(b"out", b"err")
```
But I don't think we have to mention that as attribute access is more explicit anyway and prevents one from messing up the order (i.e. `stderr, stdout = CommandOutput(...)`, after which `stdout` is stderr and `stderr` is stdout).

Reply:
Yes, good point!